ArDOS/ArdosParser/DefaultParser.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Drawing;
+using System.Text.RegularExpressions;
+using ArDOS.Model;
+using ArDOS.Parser.Exceptions;
+using ArDOS.Parser.Actions;
+
+namespace ArDOS.Parser
+{
+    public partial class WindowsParser
+    {
+        protected const RegexOptions DEFAULT_RE_OPTS = RegexOptions.Multiline | RegexOptions.Compiled | RegexOptions.IgnorePatternWhitespace;
+        public static ArdosMenu Parse(string input)
+        {
+            var sections = Regex.Split(input, @"\n^-*$\n", DEFAULT_RE_OPTS).Select(x => x.Split(new [] { "\n", "\r\n" }, StringSplitOptions.RemoveEmptyEntries)).ToArray();
+
+            // Parse title
+            var titleSection = sections[0];
+            var titleItem = ParseItem(sections[0][0], 0);
```

Comment:
You never use this. Maybe use it in the next line instead of `sections[0][0]`.

```suggestion
            var titleSection = sections[0];
            var titleItem = ParseItem(titleSection[0], 0);
```